### PR TITLE
The Great Find & Replace: to_chat() wrapper

### PR DESCRIPTION
**Pull request by ike709**
See https://github.com/tgstation/tgstation/pull/24922 for details.

Fuck multilines.

@Vivalas Have fun.

This PR is, to my knowledge, entirely ready for review. It compiles and whatnot.

Good news is, once this PR is reviewed and merged and followed by a dozen fix PRs, I can port goonchat.

Also, this isn't supposed to be used for sounds. Only text. So if you see to_chat() for a sound file, it's wrong.

🆑 ike709
tweak: Default chat output has been replaced with a to_chat() wrapper. Please report any broken or missing chat messages or sounds. This includes literally everything in the chat window, not just player messages.
/🆑

### CODE REVIEW

File: code/__DEFINES/misc.dm [hunk 1]
```diff
@@ -455,8 +455,8 @@ var/global/list/ghost_others_options = list(GHOST_OTHERS_SIMPLE, GHOST_OTHERS_DE
 #define SHELTER_DEPLOY_ANCHORED_OBJECTS "anchored objects"
 
 //debug printing macros
-#define debug_world(msg) if (Debug2) world << "DEBUG: [msg]"
-#define debug_admins(msg) if (Debug2) admins << "DEBUG: [msg]"
+#define debug_world(msg) if (Debug2) world << "DEBUG: [msg]")
+#define debug_admins(msg) if (Debug2) admins << "DEBUG: [msg]")
```
Comment (Funce) [inline code/__DEFINES/misc.dm]: The great find and replace has stuttered on this.

Comment (Funce) [inline code/__DEFINES/misc.dm]: That should fix your errors. Of Compiling

**Comment by ike709** [comment]
@Funce It actually just uncovered a shitton of other issues. But I've fixed those too.

**Comment by ike709** [comment]
It compiles locally! Woo! Haven't heard from travis yet.

**Comment by ike709** [comment]
@Vivalas The sooner this is reviewed and merged, the less conflicts I will have to resolve.

**Comment by ike709** [comment]
I've done some basic testing and this doesn't seem to fuck up chat in any obvious way.

**Comment by ike709** [comment]
Added a changelog just so people won't freak out if a random message is broken.

**Pull request merged**